stolm/HW8/HW8_2/HW8_2.py

```diff
@@ -0,0 +1,26 @@
+def check_pass(password):
```

Comment:
good job

look at this implementation

import re

def check_pass(password):
    # check length
    if not 6 <= len(password) <= 16:
        return "Your password should be between 6 and 16 characters long"

    # check for at least one uppercase letter, one lowercase letter, and one digit
    if not re.search(r"[A-Z]", password):
        return "Your password should contain at least one uppercase letter"
    if not re.search(r"[a-z]", password):
        return "Your password should contain at least one lowercase letter"
    if not re.search(r"\d", password):
        return "Your password should contain at least one digit"

    # check for at least one special character
    if not re.search(r"[$#@]", password):
        return "Your password should contain at least one of the following special characters: $ # @"

    # password is valid
    return "Your password is valid"